src/test/kotlin/org/rust/ide/inspections/import/AutoImportFixStdTest.kt

```diff
@@ -692,4 +692,44 @@ class AutoImportFixStdTest : AutoImportFixTestBase() {
 
         fn foo(x: FooBar/*caret*/) {}
     """)
+
+
+    @MockEdition(CargoWorkspace.Edition.EDITION_2018)
+    fun `test import item from workspace over std and extern crate`() = checkAutoImportFixByFileTreeWithMultipleChoice("""
+        //- dep-lib/lib.rs
+        pub mod foo {
+            pub struct AtomicUsize;
+        }
+
+        //- main.rs
+        mod bar {
+            pub struct AtomicUsize;
+        }
+
+        fn foo(t: <error descr="Unresolved reference: `AtomicUsize`">AtomicUsize/*caret*/</error>) {}
+    """, listOf("crate::bar::AtomicUsize", "dep_lib_target::foo::AtomicUsize", "std::sync::atomic::AtomicUsize"),
```

Comment:
```
ERROR: expected:<[crate::bar::AtomicUsize, dep_lib_target::foo::AtomicUsize, std::sync::atomic::AtomicUsize]> but was:<[crate::bar::AtomicUsize, dep_lib_target::foo::AtomicUsize]>
```

why `std` is not available?

Reply:
Because all `Atomic*` structs are generated by macros that are not expanded in these tests. To make it work, we use `ExpandMacros` annotation. But I suggest not to use it because it slows down test execution without actual necessity.
Let's just use another stdlib struct, for example, `std::sync::Arc`